tests/test_run_service.py

```
@@ -428,6 +428,7 @@ def test_list_runs_paginated_returns_total_and_slice(self, tmp_path, monkeypatch
 
         # Mock storage functions to use only our test runs
         monkeypatch.setattr(storage, "list_runs", lambda runs_dir=None: run_ids)
+        monkeypatch.setattr(storage, "list_all_run_ids", lambda runs_dir=None: run_ids)
```

Ruff (0.15.2)

[warning] Unused lambda argument: `runs_dir`

(ARG005)